diff -up pim-data-exporter-19.07.80/gui/org.kde.pimdataexporter.desktop.omv~ pim-data-exporter-19.07.80/gui/org.kde.pimdataexporter.desktop
--- pim-data-exporter-19.07.80/gui/org.kde.pimdataexporter.desktop.omv~	2019-07-27 19:20:02.340907595 +0200
+++ pim-data-exporter-19.07.80/gui/org.kde.pimdataexporter.desktop	2019-07-27 19:20:10.477861910 +0200
@@ -44,7 +44,7 @@ Comment[sv]=Export av inställningar fö
 Comment[uk]=За допомогою програми для експортування параметрів системи особистих даних ви зможете зберегти всі параметри програм для роботи з особистими даним, а потім відновити їх у інших системах.
 Comment[x-test]=xxPIM Setting Exporter allows to save all data from PIM apps and restore them in other systems.xx
 Terminal=false
-Categories=Qt;KDE;Network;Email;
+Categories=Qt;KDE;Network;Email;X-KDE-More;
 X-DocPath=pimdataexporter/index.html
 X-DBUS-StartupType=Unique
 X-DBUS-ServiceName=org.kde.pimdataexporter
